tests/test_orm.py

```
@@ -32,130 +33,147 @@
-        orm_cache.delete(user_b)
-        orm_cache.query(User).print_all()
-        queryset = orm_cache.query(User)
-        self.assertEqual(len(queryset.all()), 2)
-        self.assertEqual(queryset.first().name, "user a")
-        self.assertEqual(queryset.first().age, 20)
+        users = [
+            User(name="user a", age=20),
+            User(name="user b", age=30),
+            User(name="user c", age=40)
+        ]
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)